Fix:

error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'wait3' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'wait4' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- src/proc.c.orig	2011-06-18 05:24:06.000000000 -0500
+++ src/proc.c	2024-08-09 23:27:03.000000000 -0500
@@ -31,6 +31,8 @@
 
 #include "config.h"
 
+#include <string.h>
+
 #include "kernel.h"
 
 #ifdef HAVE_ALLOCA
--- lib/unix/wait.c.orig	2011-06-18 05:24:06.000000000 -0500
+++ lib/unix/wait.c	2024-08-09 23:53:23.000000000 -0500
@@ -28,10 +28,6 @@
  * THERE IS ABSOLUTELY NO WARRANTY FOR THIS SOFTWARE.
  */
 
-#ifdef __MACH__
-#  define _POSIX_SOURCE
-#endif
-
 #include "unix.h"
 
 #include <string.h>
